pages/main.js

```diff
@@ -888,10 +911,25 @@ tryHomepageInit();
  * Handles the scroll animations
  */
 function scrollAnimation() {
+
+  /**
+   * 
+   * @param {HTMLElement} element 
+   * @returns {boolean}
+   */
+  function elementFilter(element){
```

Comment:
Style nitpick - this code base typically adds a space between the function name and the brackets.